hooks/use-curriculums.js

```diff
@@ -0,0 +1,17 @@
+import useFetch from "./use-fetch";
+
+const useCurriculums = () => {
+  const { data, error } = useFetch(`/api/curriculums`);
+
+  if (error || !data) return undefined;
+  const curriculums = data
+    ?.map((curriculum) => ({
```

Comment:
Is the `?` necessary if earlier you are returning another result when data is falsey? I would think it isn't.